extensions/shared/external-media/sources/google-photos/google-photos-media.js

```diff
@@ -30,10 +36,15 @@ function GooglePhotosMedia( props ) {
 		allowedTypes,
 		path,
 		copyMedia,
+		showAdditionalFilters = false,
```

Comment:
I made this so we can simply bring back more filters.

Reply:
I'm unclear on the latest status of those. Did we decide to remove all of them or keep the ones that are working?

Reply:
Yeah, in the parent issue. The first version will be with a simplified UI but I didn't want to rewrite/remove all the logic for all other filters. Thus the default to `false` while keeping things around.